DevelopGitHub/GitHubExtension/GitHubExtension.Activity.Internal.WebApi/Commands/ActivityContextCommand.cs

```diff
@@ -1,8 +1,9 @@
-﻿using GitHubExtension.Activity.DAL;
+﻿using System;
+using GitHubExtension.Activity.DAL;
 
 namespace GitHubExtension.Activity.Internal.WebApi.Commands
 {
-    public class ActivityContextCommand : IActivityContextCommand
+    public class ActivityContextCommand : IActivityContextCommand, IDisposable
```

Comment:
You really needed OWIN context? OWIN in our case it's like DI as SimpleInjector. I suggest that you can you use SimpleInjector instead.